_guidelines-v4/01-introduction/01-about/02-acknowledgments.md

```diff
@@ -0,0 +1,19 @@
+---
+version: "v4"
+title: "Acknowledgments"
+sectionid: "acknowledgments"
+---
+
+Many institutions and individuals assisted in the preparation of these Guidelines and in the overall development of the Music Encoding Initiative framework.
```

Comment:
…framework and community.

Reply:
done (though I'm not sure about this one)